### DISCUSSION TITLE
Welcome to Voron-EVA-conversion Discussions!

### DISCUSSION THREAD

**Discussion by majarspeed** [majarspeed/Voron-EVA-conversion#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by K4r98** [comment]
Hey majarspeed. I printed your parts. Fits well :-) But wasn't "happy" the way to fix the belts. So I modified your Front with the kind of Voron Style. Now I don't need any cable ties. 

I am new in Gitgub. So I don't know how share pictures or files, if it's interesting. 

Best regards

**Reply by majarspeed** [reply]
I am on discord if you want to ping me on there. Dustinspeed#6423

**Comment by HMEM-17** [comment]
Great mod! Any plan to see the EVA mod for the voron Switchwire?
